### ISSUE TITLE
Ligands missing voicing name response

### ISSUE THREAD

**Issue by terracoda** [phetsims/membrane-transport#404]
Related to https://github.com/orgs/phetsims/projects/111/views/17?pane=issue&itemId=123020992&issue=phetsims%7Cmembrane-transport%7C396

The interactive ligands need a name response on focus.

more to come for mouse input.

**Comment by brettfiedler** [comment]
Currently we say the name when we pick up a protein - do the same for a ligand?
